spec/integration/amend_spec.rb

```diff
@@ -23,7 +23,7 @@
 
   before do
     VCR.use_cassette('authentication_success', vcr_options) do
-      client.authenticate!
+      client
```

Comment:
Assuming this fits in your 80 characters

`VCR.use_cassette('authentication_success', vcr_options) { client }`